integration-cli/docker_cli_run_test.go

```diff
@@ -404,8 +446,15 @@ func (s *DockerSuite) TestRunVolumesFromSymlinkPath(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunExitCode(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	_, exit, err := dockerCmdWithError("run", "busybox", "/bin/sh", "-c", "exit 72")
+	var (
+		exit int
+		err  error
+	)
+	if daemonPlatform == "windows" {
+		_, exit, err = dockerCmdWithError("run", WindowsBaseImage, "cmd", "/s", "/c", "exit 72")
```

Comment:
Indeed.
Aren't we really testing here that the daemon sets the expected exit code and the client exits with said exit code rather than a specific feature of cmd or sh?
Basically the reason is to keep the test as simple as possible, if we can make it work with the same command on both platforms that's ideal.